xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecAndroidMediaCodec.cpp

```diff
@@ -755,7 +756,30 @@ bool CDVDVideoCodecAndroidMediaCodec::Open(CDVDStreamInfo &hints, CDVDCodecOptio
       // Unsupported type?
       xbmc_jnienv()->ExceptionDescribe();
       xbmc_jnienv()->ExceptionClear();
-      continue;
+
+      // There is some confusion for video/VC1 vs. video/wvc1 especially on Sony devices, while
+      // JANA defines VC-1 as video/vc1, Android API defines it as video/wvc1
+      bool isVC1 = StringUtils::Contains(m_codecname, "vc1", false);
+      if (isVC1)
```

Comment:
```c++
if (!isVC1)
  continue;
```

.. to safe some curly braces and `else`. ;-)

Reply:
Will do :-) - but not sure. I think I superseed this one, with the more generic fix.